src/Components/MushroomCard/MushroomCard.js

```diff
@@ -3,22 +3,36 @@ import "./MushroomCard.css";
 import ToggleSave from '../ToggleSave/ToggleSave';
 import { formatProbability } from "../../utils";
 import { useState } from "react";
+import { useLocation } from 'react-router-dom'
```

Comment:
You're on fire! Way to dig into your tool bank & implement a simple fix to our probability issue!